### PR TITLE
Update release 1.7.0

### PR DESCRIPTION
**Pull request by mscheltienne**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mscheltienne** [comment]
@conda-forge-admin, please rerender

**Comment by mscheltienne** [comment]
@conda-forge-admin, please rerender

**Comment by mscheltienne** [comment]
@conda-forge-admin, please rerender

**Comment by mscheltienne** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
+  skip: true  # [py<310]
   entry_points:
     - mne-lsl = mne_lsl.commands.main:run
 
 requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [linux]
```
Comment (mscheltienne) [inline recipe/meta.yaml]:
```suggestion
    - make
```

**Comment by mscheltienne** [comment]
@conda-forge-admin, please rerender

**Comment by mscheltienne** [comment]
@larsoner Merging this for now, I'm not entirely sure how to add the arm64 build.. any idea?

**Comment by larsoner** [comment]
Should be able to open a PR like this, then the autotick bot should kick off a migration PR I think

https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5973

**Comment by mscheltienne** [comment]
Done here: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6728

Let's see.

**Pull request merged**